src/mach/load_command.rs

```diff
@@ -998,13 +1000,53 @@ pub struct EncryptionInfoCommand64 {
 
 pub const SIZEOF_ENCRYPTION_INFO_COMMAND_64: usize = 24;
 
+/// An enumeration of platforms currently identifiable within a version_min_command.
+#[non_exhaustive]
+#[derive(Debug)]
+pub enum Platform {
```

Comment:
If these are tied to the LC values why not make it repr(u32) and set the values to the constants ? Maybe you can’t do that with constants in enums can’t remember

Reply:
I thought you couldn't either, but I just tried and it works :shrug:. So I'll remove the trait usage in favour of `repr(u32)`.